src/harness/cbsd_sas_interface.py

```diff
@@ -0,0 +1,167 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
```

Comment:
It was moved using git mv:
a13173847bc3a02a58043c727223fbb123481df5
Renamed: sas_interface.py to cbsd_sas_interface.py